collections/_article/godot-showcase-dogwalk.md

```diff
@@ -0,0 +1,94 @@
+
+## Were there any quirks with the way the material rendering and animations were converted from Blender to Godot?
+
+<strong class="simon">Simon:</strong>
+We actually ran into a couple of issues with our stop motion animation style, which apparently seems to not be very common for game animation. So both on the export and the import level there were some parts that made assumptions over keyframe interpolation causing some obscure issues that took quite some investigation. But now those are all resolved.
+Since Godot was always our target for rendering, there were some shader features that we didn’t replicate for the preview that we had in Blender. That’s mainly the fake paper thickness, for which we had two separate effects. But since the effects are so subtle, it wasn’t really required for the asset creation process.
+We’re also not used to having to make drastic cuts due to performance, coming from usually doing offline rendering. So there were some rendering features that we had to disable and replace with some clever lighting cheats to save the frames.
+
+![](/assets/showcase/dogwalk/dogwalk-8.jpg)
+
+## Are there suggestions for improvement in this context?
+
+<strong class="simon">Simon:</strong>
+Most of the things we ran into, we reported right away and they have already been fixed. It’s been great to see how actively the Godot community has been responding to the issues we found.
+There have been some other more big picture issues that we ran into when collaborating as a team and using versioning. This mainly relates to UIDs and caching going out of sync.
+Since these issues are usually complex and relate to our pipeline and people’s individual systems, these were a bit more tricky to identify and report. Especially with the additional burden of our art team using git for the first time, it wasn’t always clear what were issues that were actually caused by Godot.
+I really hope that we can find the time to sit together with some Blender and Godot developers to see what we can collectively take away from these issues and potentially change to make sure collaborating as a team using Blender and Godot is a smooth process.
+
+## While making your game, were there any features that were sorely missing in Blender or in Godot?
+
+<strong class="julien">Julien:</strong>
+Nothing that couldn’t be added manually or fixed with workarounds. 
+With one of the main gameplay and visual elements being the leash, sadly there was no Line3D node. But luckily the community had a [handy plugin](https://github.com/CozyCubeGames/godot-lines-and-trails-3d ) to immediately fill in that gap.
+We also ran into some limitations with our animation system, but that [will be fixed in the upcoming release](https://github.com/godotengine/godot/pull/102398#issuecomment-2949711987).
```

Comment:
```suggestion
We also ran into some limitations with our animation system, but that [will be fixed in the upcoming 4.5 release](https://github.com/godotengine/godot/pull/102398#issuecomment-2949711987).
```

alternatively:

```suggestion
We also ran into some limitations with our animation system, but that [will be fixed in an upcoming release](https://github.com/godotengine/godot/pull/102398#issuecomment-2949711987).
```

Because otherwise "the upcoming release" is a bit too vague.